plugins/codescene/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@backstage/plugin-codescene",
+  "version": "0.1.0",
```

Comment:
Recent quirk, but plugins should now start at `0.0.0`, or the first published version will end up being `0.2.0`

Reply:
👍